### PR TITLE
Made examples runable

### PR DESCRIPTION
**Pull request by Himura2la**
When I tried to start an example with the non-default `num run start`, it was complaining that the `withViewModel` is not a function of something similar.

I've updated the `package.json` of two examples with the current version generated by the `create-resolve-app`.

Now they are runable in just two intuitive commands:
```
npm i
npm run dev
```

### CODE REVIEW

File: examples/todo-two-levels/package.json [hunk 1]
```diff
@@ -3,25 +3,30 @@
...
+    "resolve-scripts": "0.2.1",
+    "resolve-storage-lite": "^0.2.1"
   },
   "scripts": {
-    "start": "resolve-scripts dev",
+    "build": "resolve-scripts build",
+    "dev": "resolve-scripts dev",
+    "start": "resolve-scripts start",
     "update": "resolve-scripts update",
+    "flow": "flow",
```
Comment (IhostVlad) [inline examples/todo-two-levels/package.json]: Examples include only functional tests on `testcafe`, so `flow` or `jest` invocation will raise an error

File: examples/todo/package.json [hunk 2]
```diff
@@ -3,23 +3,28 @@
...
+    "react-dom": "^16.2.0",
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
-    "resolve-bus-memory": "^0.1.0",
-    "resolve-redux": "^0.1.0",
-    "resolve-scripts": "^0.1.0",
-    "resolve-storage-lite": "^0.1.0"
+    "resolve-bus-memory": "^0.2.1",
+    "resolve-redux": "^0.2.1",
+    "resolve-scripts": "0.2.1",
```
Comment (IhostVlad) [inline examples/todo/package.json]: An typo or should be fixed version?

File: examples/todo-two-levels/package.json [hunk 3]
```diff
@@ -3,25 +3,30 @@
...
-    "resolve-redux": "^0.1.0",
-    "resolve-scripts": "^0.1.0",
-    "resolve-storage-lite": "^0.1.0"
+    "resolve-bus-memory": "^0.2.1",
+    "resolve-redux": "^0.2.1",
+    "resolve-scripts": "0.2.1",
+    "resolve-storage-lite": "^0.2.1"
   },
   "scripts": {
-    "start": "resolve-scripts dev",
```
Comment (IhostVlad) [inline examples/todo-two-levels/package.json]: Very good proposition! But now **CI** perform `start` script invocation, so `Jenkinsfile` should be changed too: here https://github.com/reimagined/resolve/blob/v0.2.2/examples/todo/tests/functional/testcafe_runner.js#L27 and here https://github.com/reimagined/resolve/blob/v0.2.2/examples/todo-two-levels/tests/functional/testcafe_runner.js#L27

File: examples/todo/package.json (hunk 2 above)
Comment (Himura2la) [reply examples/todo/package.json]: This is a straight copypasted from the file generated by `create-resolve-app`. If the version is incorrect, we should check the code that generates it

**Comment by Himura2la** [comment]
Should I create another PR or this one can be redirected to another branch?

**Comment by IhostVlad** [comment]
> Himura2la changed the base branch from master  to v0.2.2  12 seconds from now

Yes, it's right

File: examples/todo/package.json (hunk 2 above)
Comment (IhostVlad) [reply examples/todo/package.json]: Okay, will take it in ming

File: examples/todo-two-levels/package.json (hunk 3 above)
Comment (Himura2la) [reply examples/todo-two-levels/package.json]: done

**Comment by Himura2la** [comment]
@IhostVlad, May I ask you to merge it if it's OK now? I'm still not comfortable with the conventions I never used to follow...

**Pull request merged**